src/xbot/common/controls/actuators/XCANTalon.java

```diff
@@ -129,4 +130,5 @@
     // Custom helpers -------------------------------------
     void createTelemetryProperties(String deviceName, XPropertyManager propertyManager);
     void updateTelemetryProperties();
+    void ensureModeForTalon(TalonControlMode mode);
```

Comment:
Given that this is in the `XCANTalon` class, I think we can drop the `ForTalon`.